app/assets/javascripts/components/miq_ae_class/namespace-form.js

```diff
@@ -48,8 +48,12 @@ ManageIQ.angular.app.component('namespaceForm', {
 
     vm.cancelClicked = function() {
       miqService.sparkleOn();
-      var type = vm.aeNsDomain ? __("Domain") : __("Namespace");
-      var message = vm.newRecord ? sprintf(__('Add of %s \"%s\" cancelled by user.'), type, vm.namespaceModel.name) : sprintf(__('Edit of %s \"%s\" cancelled by user.'), type, vm.namespaceModel.name);
+      if (vm.aeNsDomain) {
+        var flash_str = vm.newRecord ? __('Add of Domain "%s" cancelled by user.') : __('Edit of Domain "%s" cancelled by user.');
+      } else {
+        var flash_str = vm.newRecord ? __('Add of Namespace "%s" cancelled by user.') : __('Edit of Namespace "%s" cancelled by user.');
```

Comment:
`Add of Namespace %s` - I don't see how this is useful? Naming a nonexisting namespace? Even if the use entered a name, it's not being saved, so not needed to store it.

Reply:
Removed the name from string as suggested. Köszönöm-ió.

Reply:
Yep, added back. Köszönöm-ió.